swagger.yaml

```diff
@@ -5479,6 +5480,19 @@ definitions:
       initrd_url:
         type: string
         description: URL address to download the initrd.
+      host_fs_mount_dir:
+        type: string
+        description: The base directory on the host that contains the /boot folder.
```

Comment:
what is this directory for? if the agent is going to download the artifacts into that directory when this request is sent, I think the description should mention that

Reply:
Thank you for the feedback! I've added that to the description